fdbbackup/FileDecoder.actor.cpp

```diff
@@ -690,19 +805,33 @@ ACTOR Future<Void> decode_logs(Reference<DecodeParams> params) {
 	BackupDescription desc = wait(container->describeBackup());
 	std::cout << "\n" << desc.toString() << "\n";
 
-	state std::vector<LogFile> logs = getRelevantLogFiles(listing.logs, params);
-	printLogFiles("Relevant files are: ", logs);
+	state std::vector<LogFile> logFiles = getRelevantLogFiles(listing.logs, params);
+	printLogFiles("Relevant log files are: ", logFiles);
+
+	state std::vector<RangeFile> rangeFiles = getRelevantRangeFiles(listing.ranges, params);
```

Comment:
Can you add a new command line option to selective decode 1) log files; 2) range files; and 3) both log and range files?

Reply:
added -t log|range|both option, with default as both.